rotkehlchen/chain/ethereum/modules/sushiswap/constants.py

```diff
@@ -1,4 +1,3 @@
 from typing import Final
 
-CPT_SUSHISWAP_V1: Final = 'sushiswap-v1'
```

Comment:
sushiswap v1 has been deprecated totally?

Asking because a search didn't return anything useful.

Reply:
The string was there but was not being used. Not sure who added it. Also there is no versioning afaik like uniswap has